docs/testing/stencil-testrunner/02-config.md

```diff
@@ -16,5 +16,5 @@ Keep in mind that the usual way of configuring Jest (`package.json` and `jest.co
 Some additional Stencil specific options may be set here as well for configuring the e2e tests.
 
 ```tsx reference title=""
-https://github.com/ionic-team/stencil/blob/278c336166e27d329fffb686f2ecb3185d156696/src/declarations/stencil-public-compiler.ts#L1817-L1963
+https://github.com/ionic-team/stencil/blob/d847e92fbc297754cb8dbb7f633de9ce906f54ac /src/declarations/stencil-public-compiler.ts#L1892-L2044
```

Comment:
Giving a 404 I think cuz of the space in the URL

```suggestion
https://github.com/ionic-team/stencil/blob/d847e92fbc297754cb8dbb7f633de9ce906f54ac/src/declarations/stencil-public-compiler.ts#L1892-L2044
```

Reply:
We'd have to update this SHA after we land the feature right? Otherwise, what happens when that commit gets cleaned up by GitHub?

Reply:
Ups, removed the space. 

> We'd have to update this SHA after we land the feature right?

No need for that. GitHub keeps references to all SHAs around forever. While it might feel weird to use a commit sha that is not part of `main` it should't matter for this use case as every commit remains on GitHub once pushed.